detection-rules/callback_phishing_phone_number_in_subject.yml

```diff
@@ -0,0 +1,70 @@
+name: "Callback Phishing phone number in subject"
+description: |
+  This rule inspect messages with no attachments, from first-time senders,
+  that contain phone number patterns in the subject and specific keywords associated
+  with callback phishing brands.
+type: "rule"
+severity: "medium"
+source: |-
+  type.inbound
+  and length(attachments) == 0
+  and (
+    (
+      sender.email.domain.root_domain in $free_email_providers
+      and sender.email.email not in $recipient_emails
+    )
+    or (
+      sender.email.domain.root_domain not in $free_email_providers
+      and sender.email.domain.domain not in $recipient_domains
+    )
+  )
+  and (
+    regex.contains(subject.subject, '\+\d{1,3}[0-9]{10}')
+    or // +12028001238
+   regex.contains(subject.subject, '[0-9]{3}\.[0-9]{3}\.[0-9]{4}')
+    or // 202.800.1238
+   regex.contains(subject.subject, '[0-9]{3}-[0-9]{3}-[0-9]{4}')
+    or // 202-800-1238
+   regex.contains(subject.subject, '\([0-9]{3}\)\s[0-9]{3}-[0-9]{4}')
+    or // (202) 800-1238
+   regex.contains(subject.subject,
+                  '\([0-9]{3}\)-[0-9]{3}-[0-9]{4}'
+    ) // (202)-800-1238
+  )
+  and (
+    any([subject.subject],
```

Comment:
this in-line list isn't necessary. unless you meant to combine it with the below?